scenes/enumerated/problem_12629/problem_12629.py

```diff
@@ -0,0 +1,331 @@
+
+        arrow1 = Arrow(start=task1[0].get_center()+np.array([0,-.1,0]), end=new_property_1[0], buff=.1)
+        arrow2 = Arrow(start=task1[-2], end=new_property_1[-4], buff=.1)
+        self.play(GrowArrow(arrow1), GrowArrow(arrow2))
+        self.wait()
+        self.play(
+            task1[0].animate.set_color(ORANGE),
+            task1[-2].animate.set_color(ORANGE),
+            new_property_1[0].animate.set_color(ORANGE),
+            new_property_1[3].animate.set_color(ORANGE),
+            new_property_1[6].animate.set_color(ORANGE)
+        )
+        self.wait()
+        self.play(
+            Indicate(task1[0], rate_func=rate_functions.there_and_back_with_pause),
+            Indicate(task1[-2], rate_func=rate_functions.there_and_back_with_pause),
+            Indicate(new_property_1[0], rate_func=rate_functions.there_and_back_with_pause),
+            Indicate(new_property_1[3], rate_func=rate_functions.there_and_back_with_pause),
+            Indicate(new_property_1[6], rate_func=rate_functions.there_and_back_with_pause),
+            run_time=2
+        )
+        self.wait()
+        self.replace_items_in_formula(new_property_1, [0, 3, 6], ['x', 'x', 'x'])
+        self.play(FadeOut(arrow1), FadeOut(arrow2))
+        self.wait()
+
+        arrow4 = Arrow(start=task1[1], end=new_property_1[1].get_center(), buff=.2)
+        arrow4 = Arrow(start=task1[1], end=new_property_1[7].get_center(), stroke_width=3.5, max_tip_length_to_length_ratio=0.1, buff=.2)
+        self.play(
+            GrowArrow(arrow4),
+            GrowArrow(arrow4)
+        )
+        self.wait()
+        self.play(
+            task1[1].animate.set_color(GREEN),
+            new_property_1[1].animate.set_color(GREEN),
+            new_property_1[7].animate.set_color(GREEN)
+        )
+        self.wait()
+        self.play(
+            Indicate(task1[1], rate_func=rate_functions.there_and_back_with_pause),
+            Indicate(new_property_1[1], rate_func=rate_functions.there_and_back_with_pause),
+            Indicate(new_property_1[7], rate_func=rate_functions.there_and_back_with_pause),
+            run_time=2
+        )
+        self.wait()
+        self.play(ReplaceItemsInFormula(new_property_1, [1, 7], ['^3', '^3']))
+        self.play(FadeOut(arrow4, arrow4))
+        self.wait()
+
+        arrow5 = Arrow(start=task1[-1].get_center()+np.array([0, .1, 0]), end=new_property_1[-2], buff=0.3)
+        self.play(
+            task1[-1].animate.set_color(ORANGE)
+        )
+        self.play(Indicate(task1[-1], rate_func=rate_functions.there_and_back_with_pause, run_time=2))
+        self.play(
+            AnimationGroup(
+                GrowArrow(arrow5, rate_func=rate_functions.there_and_back_with_pause, run_time=4),
+                AnimationGroup(
+                    FadeIn(SurroundingRectangle(new_property_1[-3:], color=BLUE, corner_radius=.2), rate_func=rate_functions.there_and_back_with_pause, run_time=4),
+                    AnimationGroup(
```

Comment:
How does this `AnimationGroup` help?

Reply:
էս մասը պետքա դզեի մոռացել եմ, էսքան շատ `AnimationGroup`-ի կարիք չկա